docs/source-2.0/guides/model-translations/generating-cloudformation-resources.rst

```diff
@@ -44,48 +44,105 @@ Generating Schemas with smithy-build
-package can be used with smithy-build and the `Smithy Gradle plugin`_ to
+package can be used with either the `Smithy Gradle plugin`_ or :ref:`Smithy CLI <smithy-cli>` to
 generate CloudFormation Resource Schemas from Smithy models.
 
-The following example shows how to configure Gradle to generate CloudFormation
-Resource Schemas from a Smithy model :ref:`using a buildscript dependency
-<artifacts-from-smithy-models>`:
+.. tab:: Smithy CLI
 
-.. code-block:: kotlin
-    :caption: build.gradle.kts
-    :name: cfn-smithy-build-gradle
+    The following example shows how to generate CloudFormation
+    Resource Schemas from a Smithy model using the Smithy CLI:
 
-    plugins {
-        java
-        id("software.amazon.smithy").version("0.6.0")
-    }
+    .. code-block:: json
+        :caption: smithy-build.json
 
-    buildscript {
-        dependencies {
-            classpath("software.amazon.smithy:smithy-aws-cloudformation:__smithy_version__")
+        {
+            "version": "1.0",
+            "maven": {
+                "dependencies": [
+                  "software.amazon.smithy:smithy-aws-cloudformation:__smithy_version__",
+                  // Required for @service trait
+                  "software.amazon.smithy:smithy-aws-traits:__smithy_version__"
+                ]
+            },
+            "plugins": {
+                "cloudformation": {
+                    "service": "smithy.example#Queues",
+                    "organizationName": "Smithy"
+                }
+            }
         }
-    }
 
-The Smithy Gradle plugin relies on a ``smithy-build.json`` file found at the
-root of a project to define the actual process of generating the CloudFormation
-Resource Schemas. The following example defines a ``smithy-build.json`` file
-that generates a CloudFormation Resource Schemas for the specified resource
-shapes bound to the ``smithy.example#Queues`` service using the ``Smithy``
-organization.
+    .. important::
 
-.. code-block:: json
-    :caption: smithy-build.json
-    :name: cfn-smithy-build-json
+        A dependency on "software.amazon.smithy:smithy-aws-cloudformation:__smithy_version__" is
+        required in order for smithy-build to map the "cloudformation" plugin name
+        to the correct Java library implementation.
 
-    {
-        "version": "1.0",
-        "plugins": {
-            "cloudformation": {
-                "service": "smithy.example#Queues",
-                "organizationName": "Smithy"
+.. tab:: Gradle
+
+    The following example shows how to configure Gradle to generate CloudFormation
+    Resource Schemas from a Smithy model :ref:`using a build-only dependency
+    <artifacts-from-smithy-models>`:
+
+    .. tab:: Kotlin
+
+        .. code-block:: kotlin
+            :caption: build.gradle.kts
+
+            plugins {
+                java
+                id("software.amazon.smithy.gradle.smithy-jar").version("0.10.0")
```

Comment:
It might be nice to have a substitution for the gradle plugin versions that we can change when needed, instead of having to do a find-and-replace-all. But, if it makes the doc-build longer, maybe not worth.

Reply:
I will add this as a separate PR.